docs/content/any/getting-started/_index.md

```diff
@@ -8,3 +8,16 @@ any: false
 # Jump in
 
 Learn the basics of using Oso and get Oso set up in your application.
+
+{{% ifLang "python" %}}
+{{< callout "Early access to the next version of roles" "blue" >}}
+
+We just released early access to the next version of built-in support
+for Roles. Initially supporting SQLAlchemy, this library, includes
```

Comment:
```suggestion
for Roles. Initially supporting SQLAlchemy, this library includes
```

Also `Initially supporting SQLAlchemy, this library...` feels like an unnecessarily oblique way to indicate that support for the new roles feature is only in the `sqlalchemy-oso` library.

Reply:
Yeah, I actually agree. Originally these snippets said 

> "Using SQLAlchemy?"

> We just released early access to the next version of our SQLAlchemy library, includes improved support for common authorization models, like role based access control.

Which IMO is clearer. Maybe I should just update the title but keep the contents the same?

The goal behind these changes (from @gneray) was to make roles the headline instead of SQLAlchemy for the early access program so that people interested in other languages maybe look at it and ask for support.